README.md

```diff
@@ -168,6 +170,35 @@ produces `terminal.png`:
 <!-- generated by script/regen-screenshots.sh -->
 ![play-msgcat-font-example](doc/images/screenshot-msgcat-font.png)
 
+## Toolkits
+
+You can specify a toolkit. Available options are "java2d" (the default) and "skia".
+
+### Java2d
+
+The java2d toolkit will use swing for windowing, graphics, and events.
+
+### Skia
+
+The skia toolkit will use [skia](https://skia.org/) for graphics and [glfw](https://www.glfw.org/) for windowing and events. Using the skia toolkit requires the following extra dependency:
+
+M1 mac:
+```clojure
+com.phronemophobic.membrane/skialib-macosx-aarch64 {:mvn/version "0.9.31.0-beta"}
+```
+
+Non-M1 mac:
+```clojure
+com.phronemophobic.membrane/skialib-macosx-x86-64 {:mvn/version "0.9.31.0-beta"}
+```
+
+Linux
+```clojure
+com.phronemophobic.membrane/skialib-linux-x86-64 {:mvn/version "0.9.31.0-beta"}
+```
+
+If running from the skia project, you can use the `:skia` alias to add the dependency.
```

Comment:
Did you mean to say "if running from the membrane.term project"?

Perhaps dropping an example here would be helpful, something like:
```bash
clojure -M:membrane.term:skia run-term --toolkit skia
```